### DISCUSSION TITLE
[@types/three] Missing BufferGeometryUtils functions

### DISCUSSION THREAD

**Discussion by bergwerf** [DefinitelyTyped/DefinitelyTyped#60617] [Issues with a @types package] [answered]
Since `Geometry` was depricated I specifically need `BufferGeometryUtils.toTrianglesDrawMode`, but it seems more properties are missing.

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment, marked as answer]
The three.js types are hosted at https://github.com/three-types/three-ts-types, so please create an issue there if this is still an issue. It seems like `toTriangleDrawMode` is in the types [here](https://github.com/three-types/three-ts-types/blob/master/types/three/examples/jsm/utils/BufferGeometryUtils.d.ts#L16)?

**Reply by bergwerf** [reply]
Good point. My linter refers me to https://github.com/three-types/three-ts-types/blob/master/types/three/src/core/InstancedBufferAttribute.d.ts#L7 which is different. But `BufferGeometryUtils` is a module that must be loaded differently, so I have to figure out how to do that.

**Reply by bergwerf** [reply]
It turns out I can `import { toTrianglesDrawMode } from 'three/examples/jsm/utils/BufferGeometryUtils'`. Thanks!
